src/main/java/main/charts/TestJson.java

```diff
@@ -0,0 +1,59 @@
+package main.charts;
+import main.charts.OperationJSONUtente;
+
+public class TestJson {
```

Comment:
you shouldn't put Test code on the main package, it should be on the test package.